# https://github.com/curl/curl/pull/15026

---

--- a/lib/curl_trc.h
+++ b/lib/curl_trc.h
@@ -226,6 +226,12 @@ static void Curl_trc_smtp(struct Curl_ea
   (void)data; (void)fmt;
 }
 #endif
+#if defined(USE_WEBSOCKETS) && !defined(CURL_DISABLE_HTTP)
+static void Curl_trc_ws(struct Curl_easy *data, const char *fmt, ...)
+{
+  (void)data; (void)fmt;
+}
+#endif
 
 #endif /* !defined(CURL_DISABLE_VERBOSE_STRINGS) */
 
