--- a/libsrc/usbip_common.h
+++ b/libsrc/usbip_common.h
@@ -18,11 +18,11 @@
 #include <linux/usbip.h>
 
 #ifndef USBIDS_FILE
-#define USBIDS_FILE "/usr/share/hwdata/usb.ids"
+#define USBIDS_FILE "/opt/share/hwdata/usb.ids"
 #endif
 
 #ifndef VHCI_STATE_PATH
-#define VHCI_STATE_PATH "/var/run/vhci_hcd"
+#define VHCI_STATE_PATH "/opt/var/run/vhci_hcd"
 #endif
 
 #define VUDC_DEVICE_DESCR_FILE "dev_desc"
--- a/src/usbipd.c
+++ b/src/usbipd.c
@@ -56,7 +56,7 @@
 
 #define MAIN_LOOP_TIMEOUT 10
 
-#define DEFAULT_PID_FILE "/var/run/" PROGNAME ".pid"
+#define DEFAULT_PID_FILE "/opt/var/run/" PROGNAME ".pid"
 
 static const char usbip_version_string[] = PACKAGE_STRING;
 
